pepper-apis/src/main/java/org/broadinstitute/ddp/route/PatchProfileRoute.java

```diff
@@ -56,10 +56,8 @@ public Object handle(Request request, Response response) throws Exception {
         boolean providedLastName = json.has(Profile.LAST_NAME);
         String lastName = providedLastName ? payload.getLastName() : null;
 
-        boolean providedBirthDate = json.has(Profile.BIRTH_YEAR)
-                && json.has(Profile.BIRTH_MONTH)
-                && json.has(Profile.BIRTH_DAY_IN_MONTH);
-        LocalDate birthDate = providedBirthDate ? parseBirthDate(payload) : null;
+        boolean providedBirthDate = json.has(Profile.BIRTH_DATE);
```

Comment:
What's going on here? The old properties were marked as deprecated, not removed. We should still be supporting the old method.

Reply:
hmm. 
I guess I need to accept both types (3 date elements or full date) and act accordingly. 
Profile.BIRTH_DATE trumps deprecated elements. 
will make needed changes

Reply:
updated to be backward compatible to deprecated elements.